test/helpers/config-store.test.js

```diff
@@ -221,24 +221,3 @@ test('saves config to disk - invalid config', async (t) => {
 
     t.end();
 });
-
-test('handles no path being provided', async (t) => {
-    t.plan(1);
-    const config = new EikConfig({
-        name: 'magarita',
-        server: 'http://server',
-        files: { '/': 'pizza' },
-        version: '0.0.0',
-        out: 'muffins',
-    });
-
-    try {
-        configStore.persistToDisk(config);
-    } catch (e) {
-        t.equal(
-            e.message,
-            'The "path" argument must be of type string. Received undefined',
-        );
-    }
-    t.end();
-});
```

Comment:
Isn't this still possible now that typescript checking has been added? If a consuming library isn't using typescript then it won't have this check provided.

Reply:
I think the test was verifying that there was a cwd path which now defaults to process.cwd so the persistToDisk method no longer threw for this case

Reply:
Ah okay I missed that change. Good to remove this in that case.